src/models/stores/app-config-model.ts

```diff
@@ -17,7 +17,7 @@ const DocumentSpecModel = types
     properties: types.array(types.string)
   });
 
-const NavTabsAppConfigModel = types
+export const NavTabsAppConfigModel = types
```

Comment:
I think we should rename this to something like `NavTabsConfigModel` (it's not solely associated with the application any more) and move it elsewhere (e.g. `nav-tabs.ts`) given that it needs to be referenced from multiple places.